File: typescript/iniciante/1006.ts
import { blue, bold } from "./../fmt/colors";

const A = Number(prompt("Digite a primeira nota (A):")?.replace(",", "."));
const B = Number(prompt("Digite a segunda nota (B):")?.replace(",", "."));
const C = Number(prompt("Digite a terceira nota (C):")?.replace(",", "."));

const weightA = 2.0;
const weightB = 3.0;
const weightC = 5.0;

const average =
  (A * weightA + B * weightB + C * weightC) / (weightA + weightB + weightC);

console.log(blue(`MEDIA = ${bold(average.toFixed(1))}`));

File: swift/iniciante/1006.swift
import Foundation

if let inputA = readLine(), let inputB = readLine(), let inputC = readLine(),
   let A = Double(inputA.replacingOccurrences(of: ",", with: ".")),
   let B = Double(inputB.replacingOccurrences(of: ",", with: ".")),
   let C = Double(inputC.replacingOccurrences(of: ",", with: "."))
{
    let weightA = 2.0
    let weightB = 3.0
    let weightC = 5.0

    let average = (A * weightA + B * weightB + C * weightC) / (weightA + weightB + weightC)

    print(String(format: "MEDIA = %.1f", average))
}
